src/trade/look_player.js

```diff
@@ -0,0 +1,43 @@
+import { chunk_index, chunk_position } from '../chunk.js'
+
+export function look_player({ client, world, events }) {
+  events.on('state', ({ position }) => {
+    const { x, z } = position
+    const { traders_in_chunk } = world.traders
+    const x_chunks = [
+      chunk_position(x) - 1,
+      chunk_position(x),
+      chunk_position(x) + 1,
+    ]
+    const z_chunks = [
+      chunk_position(z) - 1,
+      chunk_position(z),
+      chunk_position(z) + 1,
+    ]
+    for (const i of x_chunks) {
+      for (const j of z_chunks) {
+        for (const k in traders_in_chunk.get(chunk_index(i, j))) {
+          const entity = traders_in_chunk.get(chunk_index(i, j))[k]
+          const yaw = Math.min(
+            127,
+            Math.max(
+              -128,
+              -((Math.atan2(x - entity.x, z - entity.z) / Math.PI / 2) * 255)
+            )
+          )
```

Comment:
```suggestion
          const yaw = Math.floor((-Math.atan2(x - entity.x, z - entity.z) / Math.PI) * (255 / 2))
```
Should do the same since atan2 return a value between -PI and PI

Reply:
done